api/client_rest_test.go

```diff
@@ -0,0 +1,23 @@
+package api
+
+import (
+	"bytes"
+	"testing"
+)
+
+func TestRESTGetDelete(t *testing.T) {
+	http := &FakeHTTP{}
+
+	client := NewClient(
+		ReplaceTripper(http),
+		AddHeader("Authorization", "Basic BASE64ENCODE_CLIENT_ID:CLIENT_SECRET"),
+		AddHeader("Content-Type", "text/plain"),
```

Comment:
I don't think we need to set any of these headers, since they're not actually exercised and therefore not relevant for this test

Reply:
Agreed